pandajedi/jedimsgprocessor/tape_carousel_msg_processor.py

```diff
@@ -87,7 +85,7 @@ def process(self, msg_obj, decoded_data=None):
                     tmp_log.warning('jeditaskid={0}, scope={1}, strangely, {2} out of {3} done...'.format(
                                                                             jeditaskid, scope, res, len(target_list)))
         except Exception as e:
-            err_str = 'failed to parse message object, skipped. {0} : {1}'.format(e.__class__.__name__, e)
+            err_str = 'failed to process the message, skipped. {0} : {1}'.format(e.__class__.__name__, e)
```

Comment:
@tmaeno 
Sorry but I don't quite understand how to have more relevant message.
The exception must be triggered here in order to abort the message consumption. While DBProxy methods do not throw up exceptions and one still needs to check DBProxy logs to understand root causes.

Reply:
The change looks OK as I just didn't like "parse message object" since it fails while doing something else than parsing message objects.